config/routes.rb

```diff
@@ -885,5 +885,24 @@ def healthcare_routes(window:)
     get :details
     get :actioncable
   end
+
+  # Routes for the HMIS API
+  # NOTE: current omniauthable setup doesn't play nicely with multiple models.
+  # If we need to use Okta and the HMIS API together, see https://stackoverflow.com/a/13591797
+  if ENV['ENABLE_HMIS_API'] == 'true' && !ENV['OKTA_DOMAIN'].present?
```

Comment:
This _should_ fix the last broken test. Omniauthable has issues with multiple models. I didn't bother fixing it, since we probably won't be using the HMIS API + Okta for some time, but it shouldn't be too difficult to do if we get there (SO link explains). @eanders